Ginger/GingerCoreNET/WorkSpaceLib/WorkSpace.cs

```diff
@@ -412,23 +412,33 @@ public bool OpenSolution(string solutionFolder)
                 ValueExpression.SolutionFolder = solutionFolder;
                 BusinessFlow.SolutionVariables = solution.Variables; 
                 solution.SetReportsConfigurations();
-                Solution = solution;                
-                UserProfile.LoadRecentAppAgentMapping();
-                
+                Console.WriteLine("Set report config");
+               Solution = solution;
+
+                Console.WriteLine("user profile mapping");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4707297)